torch/csrc/jit/api/object.cpp

```diff
@@ -35,6 +35,18 @@ void Object::define(const std::string& src, const ResolverPtr& resolver) {
       *type()->name(), src, resolver ? resolver : nativeResolver(), &self);
 }
 
+Object Object::copy() const {
+  Object obj(_ivalue()->compilation_unit(), type());
```

Comment:
This code is duplicated with `ivalue::Object::copy`, and should just call it instead.